app/jobs/message_relay_job.rb

```diff
@@ -4,12 +4,7 @@ class MessageRelayJob < ApplicationJob
   def perform(message)
     ActionCable
       .server
-      .broadcast "messages_for_#{message.room_id}", message: {
-        id: message.id,
-        content: message.content,
-        name: message.user.name,
-        created_at: message.created_at,
-        room_id: message.room_id
-      }
+      .broadcast "messages_for_#{message.room_id}",
```

Comment:
ok! but after you're verified user you can still snoop on other people's messages by subscribing to any room? are all rooms public in the app?